src/sage/topology/cell_complex.py

```diff
@@ -509,21 +508,22 @@ def homology(self, dim=None, base_ring=ZZ, subcomplex=None,
             {0: 0, 1: C2, 2: 0}
             sage: P.homology(reduced=False)
             {0: Z, 1: C2, 2: 0}
-            sage: P.homology(base_ring=GF(2))
+            sage: P.homology(base_ring=GF(2))                                           # optional - sage.modules sage.rings.finite_rings
             {0: Vector space of dimension 0 over Finite Field of size 2,
```

Comment:
Here I think `sage.modules` should go to the file, since (homology of ) cell complexes assumes use of `sage.modules`.

And again perhaps later we may decide to make cell complex module depend on `sage.modules`, and then we would want to remove `sage.modules` from doctests. Then it is easy to remove a file tag.

Reply:
But `cell_complex` defines many methods that have nothing to do with homology.
`GenericCellComplex` is a common base class for simplicial and other complexes defined in `sage.topology`

Reply:
I agree. OK.